docs/src/main/sphinx/connector/hive.rst

```diff
@@ -319,6 +319,58 @@ Hive connector documentation.
       - Maximum number of partitions to drop in a single query.
       - 100,000
 
+Hive native reader and writer configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following table lists native reader and writer configuration properties for
+the Hive connector.
+
+.. list-table:: Hive native reader and writer configuration
+    :widths: 35, 50, 15
+    :header-rows: 1
+
+    * - Property Name
+      - Description
+      - Default
+    * - ``csv.native-reader.enabled``
+      - Enable native reader of CSV file format. The equivalent catalog
```

Comment:
I think this reads better as "for CSV files." instead of "CSV file format." Same for below, e.g. "native reader of JSON files," etc.